membership.lua

```diff
@@ -85,8 +83,8 @@ end
 
 local function send_message(uri, msg_type, msg_data)
     checks('string', 'string', 'table')
-    local host, port = resolve(uri)
-    if not host then
+    local addr = resolve(uri)
```

Comment:
You skip error message here. Is it OK?

Reply:
Well. Actually I could log it on debug level, but not higher. Anyway, the log will be huge and barely useful. We've never encounter such a need, so let's ignore it.